app/src/test/java/org/simple/clinic/contactpatient/ContactPatientEffectHandlerTest.kt

```diff
@@ -295,4 +296,18 @@ class ContactPatientEffectHandlerTest {
     testCase.assertOutgoingEvents(CurrentFacilityLoaded(facility))
     verifyZeroInteractions(uiActions)
   }
+
+  @Test
+  fun `when load call result for appointment effect is received, then load call for result`() {
```

Comment:
We can just call it "load call result" instead of "load call for result"? 

Not important to change this PR. Can be done in a different PR.

Reply:
The name is `LoadCallResultForAppointment` and not call for result